### PR TITLE
Calvin/add user models

### LINKED ISSUE #12: User Models

### PR DESCRIPTION
**Pull request by calvind777**
adds user models and database migrations for admin and business

fix #12

### CODE REVIEW

**Comment by calvind777** [comment]
should be resolved @eric-ls

File: spec/factories/admins.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#  current_sign_in_ip     :inet
+#  last_sign_in_ip        :inet
+#  first_name             :string
+#  last_name              :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+
+FactoryGirl.define do
+  factory :admin do
```
Comment (charlesx2013) [inline spec/factories/admins.rb]: Unless you're planning to write tests, remove these files.

File: spec/models/admin_spec.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#  first_name             :string
+#  last_name              :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+
+require 'rails_helper'
+
+RSpec.describe Admin, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (charlesx2013) [inline spec/models/admin_spec.rb]: See about

File: spec/models/business_spec.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#  address                :string
+#  company_name           :string
+#  phone                  :integer
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+
+require 'rails_helper'
+
+RSpec.describe Business, type: :model do
```
Comment (charlesx2013) [inline spec/models/business_spec.rb]: Same

File: spec/factories/admins.rb (hunk 1 above)
Comment (eric-ls) [inline spec/factories/admins.rb]: what if we're writing tests later lol

**Pull request merged**